### PR TITLE
Changes for multiple periods in new design

### PR DESCRIPTION
**Pull request by lukas-zech-software**
Multiple periods have not been used in the application so far and a few adaption were necessary to work with the vanishing date container

### CODE REVIEW

File: source/yosoDatePickerDirective.es6.js [hunk 1]
```diff
@@ -429,6 +429,10 @@ module.exports = ngModule => {
                  * @param {int} timestamp Timestamp representing a date
                  */
                 function isDateSelected(timestamp) {
+                    if (scope.selectedListPeriod === null) {
```
Comment (mstein77) [inline source/yosoDatePickerDirective.es6.js]: _.isNull()

Comment (lukas-zech-software) [inline source/yosoDatePickerDirective.es6.js]: done

**Comment by spanitz** [comment]
For me ready2merge. Maybe hard to merge...

**Comment by lukas-zech-software** [comment]
Most likely this isn't necessary any more in the new version of the date picker

**Comment by spanitz** [comment]
Sorry, my fault - you're right. I already implemented similar behavior in the feature branch. For that reason PR can be closed.

**Comment by lukas-zech-software** [comment]
obsolete changes

**Pull request closed without merging**